tools/pipelines/test-real-service.yml

```diff
@@ -94,6 +97,7 @@ stages:
         poolBuild: Small
         testPackage: "@fluidframework/test-end-to-end-tests"
         testWorkspace: ${{ variables.testWorkspace }}
+        artifactBuildId: $(resources.pipeline.client.runID)
```

Comment:
neat, didn't know about [this variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/resources-pipelines-pipeline?view=azure-pipelines). :)